src/app/client/src/app/modules/shared/services/config/roles.config.json

```diff
@@ -24,7 +24,8 @@
        "collectionRole":["CONTENT_CREATOR","CONTENT_CREATION","CONTENT_REVIEWER","CONTENT_REVIEW"],
        "lessonplanRole":["CONTENT_CREATOR","CONTENT_CREATION","CONTENT_REVIEWER","CONTENT_REVIEW"],
        "contentUploadRole":["CONTENT_CREATOR","CONTENT_CREATION","CONTENT_REVIEWER","CONTENT_REVIEW"],
-       "allContentRole": ["CONTENT_CREATOR","CONTENT_CREATION","CONTENT_REVIEWER","CONTENT_REVIEW","BOOK_CREATOR"]
+       "allContentRole": ["CONTENT_CREATOR","CONTENT_CREATION","CONTENT_REVIEWER","CONTENT_REVIEW","BOOK_CREATOR"],
+       "collaboratingRole": ["CONTENT_CREATOR","CONTENT_CREATION","BOOK_CREATOR"]  
```

Comment:
cross check once with the roles if content reviewer can creator content he can be collaborator also